src/conditional-flow/boolean-conditions.js

```diff
@@ -3,7 +3,13 @@
 // if the value is false.
 function getResult (didPass) {
 
-  // TODO: write code in this function body to pass the tests
+  if (didPass==true) {
```

Comment:
You don't necessarily need to check if it's true here. Instead you could have used something like 
```js
if (didPass){
   return "Well done, you passed!";
}
```

As if statements want a boolean type, which you are creating by doing the `didPass==true`. However, as the variable is already a boolean, you can leave it out. If you wanted to use this method to check for false, you would do this:
```js
if(!didPass) ...
```

All that being said, if you didn't trust the variable and you wanted to make sure it was a boolean, you should use the `===` equivalence so ensure the variable is checked as if it were a boolean.

Reply:
I actually wrote that before! and then I thought it was wrong as I thought didPass could have taken multiple values. Which I now see was a mad misconception.

Reply:
You're right in thinking it is possible for `didPass` to be other values, however, because we know the tests aren't changing, and the tests only every send a boolean value to the function we know it's safe to use `if (didPass)`